src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.tsx

```diff
@@ -9,17 +9,31 @@ import { ReactComponent as ChevronLeft } from "../../../../assets/chevron-left.s
 import { useForm } from "react-hook-form";
 import AmountInput from "../../AmountInput/AmountInput";
 
+export interface ConfirmSendModalProps {
+  address: string;
+  invoiceAmount: number;
+  back: () => void;
+  balance: number;
+  close: (confirmed: boolean) => void;
+  comment: string;
+  fee: string;
+  ln: boolean;
+}
+
 const ConfirmSendModal: FC<ConfirmSendModalProps> = (props) => {
   const { t } = useTranslation();
   const appCtx = useContext(AppContext);
-  const { ln, invoiceAmount, address, fee, comment } = props;
+  const { address, back, balance, close, comment, fee, invoiceAmount, ln } =
+    props;
 
   const [amount, setAmount] = useState(0);
 
   const amountChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
     setAmount(+event.target.value);
   };
 
+  const isInvoiceAmountBiggerThanBalance = invoiceAmount > balance;
```

Comment:
Should this be an effect? Not too familiar with react in this case

Reply:
Shouldn't be an issue; ConfirmSend won't re-render that much IMO.